src/style.css

```diff
@@ -1,3 +1,76 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  --global-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__items {
+  margin-left: 20px;
+}
+
+.nav__items:first-child {
+  margin-left: 0;
+}
+
+.nav__link:link {
+  position: relative;
+  display: flex;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  font-weight: 500;
+  height: 60px;
+  align-items: center;
+  cursor: pointer;
+  transition: color 0.3s;
+}
+
+.nav__link:visited {
+  color: #000;
```

Comment:
This color also should go to variables

JFYI - in practice various link styles like visited color are set via reset.css (normalize.css) - but no need to fix it now :smile: